Description: Avoid use of Node.js module v8-compile-cache
Forwarded: no
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2022-01-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/bin/eslint.js
+++ b/bin/eslint.js
@@ -9,9 +9,6 @@
 
 "use strict";
 
-// to use V8's code cache to speed up instantiation time
-require("v8-compile-cache");
-
 //------------------------------------------------------------------------------
 // Helpers
 //------------------------------------------------------------------------------
--- a/package.json
+++ b/package.json
@@ -77,8 +77,7 @@
     "semver": "^6.1.2",
     "strip-ansi": "^5.2.0",
     "strip-json-comments": "^3.0.1",
-    "text-table": "^0.2.0",
-    "v8-compile-cache": "^2.0.3"
+    "text-table": "^0.2.0"
   },
   "devDependencies": {
     "@babel/core": "^7.4.3",
